website/server/controllers/api-v3/user.js

```diff
@@ -28,7 +28,29 @@ let api = {};
  * @apiGroup User
  *
  * @apiSuccess {Object} data The user object
- */
+ *
+ * @apiDescription The user profile contains data related to the authenticated user including (but not limited to);
+ * Achievements
+ * Authentications (including types and timestamps)
```

Comment:
@madpink, really sorry that I didn't send that review earlier. :(  This was part of it, in response to your question "so if the success example is abbreviated, should I remove the apiDescription?"

"No, I really like your apiDescription. It's a useful overview that describes the things that aren't obvious, with no significant technical details that might become outdated, and the "including (but not limited to)" part is great for making it clear that could be more items."

I still think the details you put in under (and including)  "@apiDescription The user profile contains data related to the authenticated user including (but not limited to)" were very useful. Linking to the model as well is great, but the brief, plain-English summary of the top level items would be much-loved by coders new to the model. It's exactly the sort of high-level overview that makes it easier to start understanding the details of the code. If you'd like to put back all the points you wrote, I think it would be great! But if you think it's too late and you'd rather not, I'll understand. :)